src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let number = 0;
```

Comment:
The variable `number` should be declared as a `const` since it is not reassigned. If you need to accumulate a value, you could directly return the calculated value in each condition.